testdata/p4_16_samples_outputs/table-entries-lpm-bmv2.p4.entries.txt

```diff
@@ -0,0 +1,71 @@
+updates {
+  type: INSERT
+  entity {
+    table_entry {
+      table_id: 33555353
+      match {
+        field_id: 1
+        lpm {
+          value: "\021"
+          prefix_len: 4
+        }
+      }
+      action {
+        action {
+          action_id: 16837978
+          params {
+            param_id: 1
+            value: "\000\013"
+          }
+        }
+      }
+    }
+  }
+}
+updates {
+  type: INSERT
+  entity {
+    table_entry {
+      table_id: 33555353
+      match {
+        field_id: 1
+        lpm {
+          value: "\022"
+          prefix_len: 8
+        }
+      }
+      action {
+        action {
+          action_id: 16837978
+          params {
+            param_id: 1
+            value: "\000\014"
+          }
+        }
+      }
+    }
+  }
+}
+updates {
+  type: INSERT
+  entity {
+    table_entry {
+      table_id: 33555353
+      match {
+        field_id: 1
+        lpm {
+          value: "\000"
```

Comment:
Similar question to a previous one about a field that is completely "don't care", and whether it should be omitted from this file, or present like this.